playbooks/roles/mattermail/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+- name: create mattermail group
+  group:
+    name: mattermail
+
+- name: create mattermail user
+  user:
```

Comment:
I think you can add `system: yes` here as well since this is really just a system account.